admin-tools/src/webapp/vm/aliases/chef_aliases_confirm_remove.vm

```diff
@@ -6,8 +6,8 @@
 	<h3>
 	$tlang.getString("alias.remove")
 	</h3>
-	#if ($alertMessage)<div class="alertMessage">$tlang.getString("alias.lert") $validator.escapeHtml($alertMessage)</div>#end
-	<div class="alertMessage">
+	#if ($alertMessage)<div class="sak-banner-error">$tlang.getString("alias.lert") $validator.escapeHtml($alertMessage)</div>#end
```

Comment:
Is "alias.lert" a valid message? Looks like a typo. I don't see that message key in the translations. There is a "alias.alert" key in some of the non-English translations. We should probably fix that for this PR.